### DISCUSSION TITLE
How to set the size of the AccordionItem?

### DISCUSSION THREAD

**Discussion by timoconnor** [themesberg/flowbite-svelte#619] [General]
How do I set the padding of AccordionItem?  I would like to change the padding around the span tag, i.e something like:
<Accordion>
   <AccordionItem class="p-2">...

Seems like it is set to p-5 by default.

**Comment by shinokada** [comment]
If the `flush` prop is set the padding is `py-5`:

```
<Accordion flush>
```

**Reply by timoconnor** [reply]
Thankyou!  that is exactly what I was after :)
